plugins/filter/time.py

```diff
@@ -83,7 +74,7 @@ def to_time_unit(human_time, unit='ms', **kwargs):
         h_time_int = int(res.group(1))
         h_time_unit = res.group(2)
 
-        h_time_unit = UNIT_TO_SHORT_FORM.get(h_time_unit, h_time_unit)
+        h_time_unit = unit_to_short_form.get(h_time_unit.rstrip('s'), h_time_unit)
```

Comment:
Crap, I forgot that one! Thanks for spotting!

Reply:
eh... what? I didn't see you forgot anything.

Reply:
I should have changed `UNIT_TO_SHORT_FORM` to `unit_to_short_form` so that the correct constants for months and years are used.

Reply:
Oh, lol, only now I realize I was looking at the wrong variable. You're right!

Reply:
the fear of missing anything. Hey, we reviewed your stuff! ;)